### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by stgraber**

### CODE REVIEW

File: lxd/api_1.0.go [hunk 1]
```diff
@@ -211,11 +211,6 @@ func api10Put(d *Daemon, r *http.Request) Response {
 			if err != nil {
 				return InternalError(err)
 			}
-		} else if key == "storage.lvm_fstype" {
```
Comment (hallyn) [inline lxd/api_1.0.go]: Why is this being removed?

File: lxd/storage_lvm.go [hunk 2]
```diff
@@ -161,15 +161,6 @@ func storageLVMSetVolumeGroupNameConfig(d *Daemon, vgname string) error {
 	return nil
 }
 
-func storageLVMSetFsTypeConfig(d *Daemon, fstype string) error {
```
Comment (hallyn) [inline lxd/storage_lvm.go]: Why is this being removed?

File: lxd/api_1.0.go (hunk 1 above)
Comment (stgraber) [inline lxd/api_1.0.go]: Because the "else" code path does the exact same.

File: lxd/storage_lvm.go (hunk 2 above)
Comment (stgraber) [inline lxd/storage_lvm.go]: Because it's no longer being called from api_1_0.go

**Pull request merged**